nncf/experimental/tensor/tensor.py

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) 2023 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Any, List, Optional, Tuple, TypeVar, Union
+
+import nncf.experimental.tensor.functions as functions  # pylint: disable=consider-using-from-import
```

Comment:
As one option to `import nncf.experimental.tensor.functions as functions` into a call function.

Reply:
done, call functions via `_call_function`